### ISSUE TITLE
Linux 5.6: zpool import fails on "undefined symbol: zpool_initialize_wait"

### ISSUE THREAD

**Issue by hoh** [openzfs/zfs#10167]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	| Ubuntu
Distribution Version	| 19.10
Linux Kernel	| [5.6.0-050600-generic](https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.6/)
Architecture	| x86_64
ZFS Version	| Master https://github.com/openzfs/zfs/commit/9a51738b60c2164822baefa17f8fdcebe9d82fbc
SPL Version	| Master https://github.com/openzfs/zfs/commit/9a51738b60c2164822baefa17f8fdcebe9d82fbc

### Describe the problem you're observing

When trying to import a ZFS pool using `zpool import` on Linux 5.6, I get the following error:
```bash
sudo zpool import tank
zpool: symbol lookup error: zpool: undefined symbol: zpool_initialize_wait
```

### Describe how to reproduce the problem

Install the [Ubuntu build of Linux 5.6](https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.6/) on Ubuntu 19.10 x86_64.

Clone this repository, change the version in `./META` and with a git tag.

Build the .deb ZFS packages within a Docker container using the following Dockerfile. Note that this procedure works fine for Linux 5.5.13.

```Dockerfile
FROM ubuntu:19.10

ENV DEBIAN_FRONTEND noninteractive

RUN apt-get update && apt-get --yes upgrade && apt-get install --yes gdebi-core build-essential apt-utils
COPY upstream-kernel /srv/upstream-kernel
RUN gdebi --non-interactive /srv/upstream-kernel/linux-headers-*_all.deb
RUN gdebi --non-interactive /srv/upstream-kernel/linux-headers-*-generic_*.deb
RUN gdebi --non-interactive /srv/upstream-kernel/linux-modules-*.deb
RUN gdebi --non-interactive /srv/upstream-kernel/linux-image-*.deb

RUN apt-get update && apt-get --yes upgrade && apt-get install --yes \
    build-essential autoconf automake libtool gawk alien fakeroot gdebi-core \
    zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev \
    libacl1-dev libaio-dev libdevmapper-dev libssl-dev libelf-dev \
    # linux-headers-$(uname -r) \
    # To enable the pyzfs packages additionally install the following:
    python3 python3-dev python3-setuptools python3-cffi git

RUN useradd --shell /bin/bash --system --create-home ubuntu

COPY zfs /home/ubuntu/zfs
RUN ls /home/ubuntu/zfs
RUN chown -R ubuntu /home/ubuntu/zfs
WORKDIR /home/ubuntu/zfs

USER ubuntu

RUN sh autogen.sh
#RUN ./configure --with-config=srpm
#RUN make -j12 pkg-utils deb-dkms
RUN ./configure
RUN make -j12
RUN make deb

USER root
CMD cp *.deb /mnt/
```

Run the container to extract the built packages:
```bash
sudo docker run --rm -ti -v (pwd)/deb:/mnt build-zfs
```

Install the packages with `dpkg`:
```bash
sudo dpkg -i zfs_0.8.4-1_amd64.deb zfs-dkms_0.8.4-1_amd64.deb libzpool2_0.8.4-1_amd64.deb libuutil1_0.8.4-1_amd64.deb libnvpair1_0.8.4-1_amd64.deb python3-pyzfs_0.8.4-1_amd64.deb
```

**Comment by behlendorf** [comment]
This appears to be a user space issue unrelated to the kernel.  The `zpool` command seems to be using the ubuntu provided version of libzfs which is a little older and doesn't contain the `zpool_initialize_wait` symbol.  Removing all of the ubuntu provided ZFS packages and rebuilding the master branch should resolve the issue.

**Comment by francoism90** [comment]
@behlendorf Does this mean OpenZFS is compatible with Linux 5.6? :)
I haven't tried to upgrade to this kernel.

**Comment by hoh** [comment]
@behlendorf thanks for your help. While I did not have any ubuntu provided ZFS package, removing all my built packages with `--purge` and reinstalling them worked. I will close this issue.

@francoism90 I haven't tried any operation (snapshot/...), but mounting a ZFS filesystem appears to work without any issue on my Linux 5.6 with OpenZFS.

**Comment by francoism90** [comment]
@hoh I'll try and wait for Linux 5.6 to be released in my repo. I still need to wait for an Intel WiFi fix that has been released in 5.6.1.

**Comment by behlendorf** [comment]
@francoism90 that 0.8.3 tag won't build with the 5.6 kernel yet, you'll need to use the master branch  for now for Linux 5./6 compatibility.
